engine/src/main/resources/org/terasology/engine/assets/shaders/blockSelection_frag.glsl

```diff
@@ -2,21 +2,24 @@
 // Copyright 2021 The Terasology Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-uniform sampler2D texture;
+uniform sampler2D tex;
```

Comment:
umm, do we have a better name for this, we can't use texture since its a reserved function.